sycl/test-e2e/lit.cfg.py

```diff
@@ -119,12 +119,17 @@
     config.available_features.add('gpu-intel-dg2')
 if lit_config.params.get('gpu-intel-pvc', False):
     config.available_features.add('gpu-intel-pvc')
+    config.available_features.add('matrix-fp16') # PVC implies the support of FP16 matrix
```

Comment:
Could you please add also matrix-tf32. This flag was added by Bing earlier, but gpu-intel-pvc was not updated accordingly.

Reply:
done!